tests/Feature/ValidateCSVTest.php

```diff
@@ -163,24 +164,25 @@ function (array $config): array {
             }
         ];
 
-        yield 'missing wikidata value' => [
+        yield 'invalid meta wikidata value' => [
```

Comment:
IIUC, this is now the same three test cases as before (except that the validation error changes a bit), but “missing wikidata value [when statement guid is present]” and “invalid meta wikidata value” have swapped places, which makes the diff more confusing IMHO… can we swap them back?